utils/rjson.hh

```diff
@@ -223,12 +223,11 @@ void set(rjson::value& base, rjson::string_ref_type name, rjson::string_ref_type
  * Will typically require an existing rapidjson::internal::TypeHelper<...>
  * to exist for the type written. 
  * 
- * (Note: order is important. Need to be after set(..., rjson::value), 
- *  otherwise the enable_if must be expanded)
+ * (Note: order is important. Need to be after set(..., rjson::value)).
```

Comment:
Is the order really important? I thought the standard guarantees that the more specific definition wins?

Reply:
@elcallio as the original author of the comment, what was the intent?

Probably for the concept it can be safely removed, and maybe even for the enable_if version.

Reply:
@elcallio ping

Reply:
@elcallio ping^2

Reply:
No sense in delaying this patch because of a wrong pre-existing comment.
@elcallio if you get back to this, and decide the comment is no longer right (or was never right?) please submit a separate patch to remove it.